lib/reek/ast/node.rb

```diff
@@ -2,9 +2,7 @@
 
 require_relative '../cli/silencer'
 
-Reek::CLI::Silencer.silently do
-  require 'parser'
-end
+require 'parser'
```

Comment:
So that problem with the parser gem was fixed upstream? How did you notice?

Reply:
I removed the silencer and noticed no extra output.

Reply:
This whole PR came from my hunch on what's going on in #1357.